src/p2p/connection_basic.cpp

```diff
@@ -174,13 +174,11 @@ connection_basic::~connection_basic() noexcept(false) {
 
 void connection_basic::set_rate_up_limit(uint64_t limit) {
 	
-	// TODO remove __SCALING_FACTOR...
-	const double SCALING_FACTOR = 2.1; // to acheve the best performance
-	limit *= SCALING_FACTOR;
+	// After deleting erroneous / superfluous sleeps, no more need to fudge with a "SCALING_FACTOR" of 2.1
```

Comment:
Comment probably not needed as future reviewers will not see the old code or know what this is referring to

Reply:
Yes, I understand your reasoning; I had similar thoughts. And still, I also felt a certain urge to write down why and how I got rid of that mysterious scaling factor. It was the same regarding the comments that I did in 3 other places "documenting" deleted code.

Anybody with a second opinion about this?

Reply:
This belongs in the commit message.

Reply:
In fact, I notice the explanation is not in the commit message, but only on github, which means it'll be lost when someone looks through the history.